src/tests.rs

```diff
@@ -0,0 +1,178 @@
+use crate::{mock::*, Error};
+use frame_support::{assert_noop, assert_ok};
+
+#[test]
+fn add_item_test(){
```

Comment:
I saw there are several tests in one test case. If each is independent, please divide it into different test cases to improve the maintenance. For example, I will split it into four other test cases in your add test case. 
1. Add the successful case
2. Error::<Test>::ItemTypeAlreadyExists
3. Error::<Test>::ItemTypeExceedMax64
4. Error::<Test>::ItemExceedMax128

Reply:
I restructured unit tests. Please review.